clever.go

```diff
@@ -225,6 +225,19 @@ type Section struct {
 	Term
 }
 
+type EventResp struct {
+	Links []Link
+	Event Event `json:"data"`
+	Uri   string
+}
+
+type Event struct {
+	Type    string
+	Created string
+	Id      string
+	Data    map[string]interface{}
```

Comment:
I think if the goal is to keep it close to the API response (as you said below) then calling it Object is fine, since that's what it's called in the API :)